ui/shared/GasSelector.qml

```diff
@@ -139,34 +144,44 @@ Item {
     ModalPopup {
         id: customNetworkFeeDialog
         title: qsTr("Custom Network Fee")
-        height: 386
+        height: 286
+        width: 400
 
         Input {
           id: inputGasLimit
           label: qsTr("Gas limit")
           text: "22000"
           customHeight: 56
           anchors.top: parent.top
+          anchors.left: parent.left
+          anchors.right: undefined
+          width: 222
```

Comment:
I'm not sure of the design and what it looks like, but I assume this Input is on the left of the parent, and there is something else on it's right "squeezing" it?
If that's so, it better and easier to just add the `anchors.right` and anchor it to the `otherItem.left` with the required margin. That way, if we ever scale the form, no need to change the width.

Reply:
Okay so I tried different variations of what you're proposing here, but once you remove the anchors on the parent, the input element behave unexpected and start to bounce off the screen or get an arbitrary with.

Rolled back to what I had now cause I can't seem to make it work you'd like it to be. Feel free to add a commit on top here that applies the needed changes